poutyne/framework/callbacks/color_formatting.py

```diff
@@ -4,6 +4,7 @@
 import warnings
 from typing import Dict, Union
 
+from poutyne import is_in_jupyter_notebook
```

Comment:
I don't think this modification should be in this PR but rather directly in dev.